### DISCUSSION TITLE
New Baram Version vs. Old Baram Version

### DISCUSSION THREAD

**Discussion by DirkDali** [nextfoam/baram#62] [General]
Hello,
In the future there will only be the new Baram version with the version number 24.XX. What will happen to the old versions 4.7?
Are they no longer being developed?
Ciao
Dirk

**Comment by kdg5424** [comment]
Hi @DirkDali This is Donggyu.

We will develop BARAM with the version Number 24.XX, 25.XX and so on. (The number means the release year.)
Therefore, we will support the newest version of BARAM and will not develop the old versions.
You can use the newest BARAM on Windows, Linux, MacOS with various applications easily.

If you have any questions in BARAM, contact to us.

Thank you.
Donggy kim.
